change/@griffel-webpack-extraction-plugin-2722342d-e7ad-42c7-b370-cc43d50217de.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Stop inlining the styles in the virtual CSS module request",
```

```suggestion
  "type": "patch",
  "comment": "chore: stop inlining the styles in the virtual CSS module request",
```

nit: it's not adding any functionality